modules/openapi-generator/src/test/resources/3_0/typescript-fetch/enum.yaml

```diff
@@ -0,0 +1,289 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Enum test
+servers:
+  - url: http://localhost:3000
+paths:
+  /fake/enum-request-ref:
+    get:
+      operationId: fake-enum-request-get-ref
+      parameters:
+        - name: string-enum
+          in: query
+          schema:
+            $ref: "#/components/schemas/StringEnum"
+        - name: nullable-string-enum
+          in: query
+          schema:
+            allOf:
+              - $ref: "#/components/schemas/StringEnum"
+            type: string
+            nullable: true
```

Comment:
Oops. https://github.com/OAI/OpenAPI-Specification/issues/1368#issuecomment-580375017
> In JSON Schema, allOf doesn't work like inheritance. It means that the value must conform independently to each of the listed subschemas, and it must also conform to any other conditions specified directly in the schema.

I'll remove this because quoted as above and I've found https://github.com/OAI/OpenAPI-Specification/issues/2244 . Generated codes are roughly usable but this yaml does not conform OpenAPI Spec, currently.